src/Mod/TechDraw/App/DrawViewPart.cpp

```diff
@@ -189,55 +177,31 @@ std::vector<TopoDS_Shape> DrawViewPart::getSourceShape2d() const
     return result;
 }
 
-
 TopoDS_Shape DrawViewPart::getSourceShape() const
 {
 //    Base::Console().Message("DVP::getSourceShape()\n");
-    TopoDS_Shape result;
     const std::vector<App::DocumentObject*>& links = getAllSources();
-    if (links.empty())  {
-        bool isRestoring = getDocument()->testStatus(App::Document::Status::Restoring);
-        if (isRestoring) {
-            Base::Console().Warning("DVP::getSourceShape - No Sources (but document is restoring) - %s\n",
-                                getNameInDocument());
-        } else {
-            Base::Console().Error("Error: DVP::getSourceShape - No Source(s) linked. - %s\n",
-                                  getNameInDocument());
-        }
-    } else {
-        result = ShapeExtractor::getShapes(links);
+    if (!links.empty())  {
+        return ShapeExtractor::getShapes(links);
     }
-    return result;
+    return TopoDS_Shape();
 }
```

Comment:
Code structure/formatting/unnested/simplicity 😍👠💋💪

```suggestion
    if (links.empty())  {
        return TopoDS_Shape();
    }
    return ShapeExtractor::getShapes(links);
```